source/guides/models/repositories.md

```diff
@@ -26,20 +26,23 @@ This architecture has several advantages:
 
 ## Interface
 
-When a class includes `Hanami::Repository`, it will receive the following interface:
-
-  * `.persist(entity)` – Create or update an entity
-  * `.create(entity)`  – Create a record for the given entity
-  * `.update(entity)`  – Update the record corresponding to the given entity
-  * `.delete(entity)`  – Delete the record corresponding to the given entity
-  * `.fetch(raw)`  – Fetch raw datasets for the given raw query string (eg. SQL)
-  * `.execute(raw)`  – Execute raw command (eg. SQL)
-  * `.all`   - Fetch all the entities from the collection
-  * `.find`  - Fetch an entity from the collection by its ID
-  * `.first` - Fetch the first entity from the collection
-  * `.last`  - Fetch the last entity from the collection
-  * `.clear` - Delete all the records from the collection
-  * `.query` - Fabricates a query object
+When a class includes `Hanami::Repository`, it will receive the following instance methods:
```

Comment:
Can we change 'it will receive the following instance methods' to 'objects will receive the following methods'? (since it's the objects that get the methods, not the class)